backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -4999,4 +4999,13 @@ databaseChangeLog:
             columns:
               - column:
                   name: result
-                  type: ${database.defaultSchemaName}.TEST_RESULT
\ No newline at end of file
+                  type: ${database.defaultSchemaName}.TEST_RESULT
+  - changeSet:
+      id: add-missing-header-to-test_result_upload_error
+      author: boban@skylight.digital
+      comment: add MISSING_HEADER to test_result_upload_error
+      changes:
+        - sql:
+            sql: |
+              ALTER TYPE ${database.defaultSchemaName}.TEST_RESULT_UPLOAD_ERROR ADD VALUE 'MISSING_HEADER';
+      rollback: ""
```

Comment:
Huh, so what exactly happens on rollback?

FWIW, how I've seen this this handled in the past is as a multi-step process:

1) Create a new enum with only the values you want to retain and give it a temporary name, e.g. `test_result_upload_error_old`
2) Drop the original enum
3) Re-name your newly-created enum to match the original name, i.e. `test_result_upload_error`
4) 💥

Reply:
Nothing happens here on roll back, the enum will continue the exist with the new value. If we want to remove that enum value won't we also have to remove all rows in the DB where test_result_upload_error = "MISSING_HEADER"? Personally I don't see a whole lot of value of adding a rollback for this, but if anyone else disagrees I can add what you suggested.

Reply:
Gotcha, we just have to specify the rollback block explicitly here because Liquibase can't generate one on its own I assume?

I'm likewise not really worried about this but thank you for clarifying!

Reply:
Yep! Liquibase requires rollbacks for all changes. SQL is one of the ones that it can't autogenerate a rollback for so this block is required.